### DISCUSSION TITLE
Bot suddenly requires slash (/) before responding to text . . . ?

### DISCUSSION THREAD

**Discussion by humblecoder** [telegraf/telegraf#1826] [Q&A]
So, I'm having a weird thing happen with Telegraf 4.9.2.  I've been using `bot.on('text'...)` since the start.  It always accepts text responses and processes them just fine.  Since a `pnpm i` earlier today (still specifying 4.9.2 explicitly), my responses require `/` before them to be accepted.

I've made no changes to bot privacy or otherwise.  I've simply made some unrelated backend updates and redeployed.  What random other thing might I have changed?  🤔

**Comment by MKRhere** [comment]
This behavior is consistent with groups and private chat with the bot?

Run your bot in debug mode (set the env var `DEBUG=telegraf:*`) send some messages, and show the logs it has received.

**Comment by humblecoder** [comment]
Hrm.  Toggled privacy `on/off/on/off` a few times.  Had no effect.  It seems that only toggling the bot to ADMIN or disinviting/reinviting "fixes" the issue.  But no idea what caused it in the first place.  

To be clear, this happened after transferring the bot to a new server in a different region.  I have no idea of TG API sensed an IP change and "locked down" or what.

Regardless, I definitely need to do some digging into the issue.

Thanks for the input.
